package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "uncrustify",
-  "version": "0.68",
+  "version": "0.68.1",
```

Comment:
Why not `.0` then?

Reply:
Can't publish to npm without the .patch version number. 
I thought the version was 0.68.1 already? (at least what I see in the Releases tab in GH and also the tags for the repo)

Reply:
My mistake then! You are right, there has been a precedent having a patch version number as well.